lib/grpc/connection.js

```diff
@@ -273,7 +276,11 @@ class GrpcConnection extends EventEmitter {
       this._metrics.getOrCreateMetric(NAMES.INFINITE_TRACING.SPAN_RESPONSE_ERROR)
         .incrementCallCount()
 
-      logger.trace('span stream error. Code: [%s]: %s', err.code, err.details)
+      // For errors, the status will either result in a disconnect or a reconnect
+      // delay that should prevent too frequent spamming. Unless the app is idle
+      // and regularly getting Status 13 reconnects from the server, in which case
+      // this will be almost the only logging.
+      logger.warn('Span stream error. Code: [%s]: %s', err.code, err.details)
```

Comment:
This is the only increased risk part of the PR. This will help with troubleshooting as these issues are not always reproducible after the fact. We have a delay before retrying, so it won't be super heavy spamming of logs but sometimes customers still don't like that. 

Also left a comment about what I learned recently with regards to idle-applications and the server sending Code 13 errors.

Reply:
Seems pretty minor still, nice knowledge share in the comment 👍